BarrysPeanutsAkkaFlat/src/main/java/barryspeanuts/actor/CustomerActor.java

```diff
@@ -0,0 +1,83 @@
+package barryspeanuts.actor;
+
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.AbstractBehavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.actor.typed.javadsl.Receive;
+import java.util.UUID;
+
+public class CustomerActor extends AbstractBehavior<Object> {
+
```

Comment:
You are inconsistent on whether you have a blank line as the first line of a class

Reply:
I have been relying on Spotless to do the formatting automagically @cretz . Nonetheless, is "no blank after the first line of the class" the convention to use?

Reply:
I am unsure of the convention tbh, just pointing out that it's inconsistent here

Reply:
addressed